docs/FAQ/multi_devices.md

```diff
@@ -148,6 +148,13 @@ Note that here we
 - `-n` number of worker nodes to run on
 - `-H` the host file which is required by `ssh` and `mpi`
 - `--kv-store` use either `dist_sync` or `dist_async`
+- If the `-s` argument is not passed, it will keep the number of servers same as number of workers
+- The launch.py script tries to cycle through the hosts file to launch the servers and workers. For eg.
+  Lets say you have `5` hosts in the hosts file and you passed n as `3`(and nothing for s).
```

Comment:
`For eg. Lets say ` -> `For example, let's say`

Reply:
Updated.